server/models/CollectionJob.js

```diff
@@ -31,9 +31,20 @@ module.exports = function (sequelize, DataTypes) {
         onDelete: 'SET NULL',
         allowNull: true,
         unique: true
+      },
+      secret: {
+        type: DataTypes.UUID,
+        allowNull: false
```

Comment:
that would lead to a single default uuid per time the app started up i think... 

The `beforeValidate` hook was the most recommended and stable feeling suggestion I ran across for dealing with default values with `uuid()`

Reply:
Oh right that makes sense. Would [Sequelize's `UUIDV4` make sense here?](https://sequelize.org/api/v7/classes/_sequelize_core.index._internal_.uuidv4)

Reply:
Sounds good. Thanks for trying!